vault/login_mfa.go

```diff
@@ -2013,11 +2015,29 @@ func (c *Core) validateTOTP(ctx context.Context, creds []string, entityMethodSec
 
 	usedName := fmt.Sprintf("%s_%s", configID, creds[0])
 
-	_, ok := c.loginMFABackend.usedCodes.Get(usedName)
+	_, ok := usedCodes.Get(usedName)
 	if ok {
 		return fmt.Errorf("code already used; new code is available in %v seconds", totpSecret.Period)
 	}
 
+	numAttempts, _ := usedCodes.Get(configID)
```

Comment:
what does configID represent? is it unique per MFAmethod/user?

Reply:
configID is the methodID of MFA method.

Reply:
I think this is the MFA method id (UUID), unique per MFA method, unrelated to user.

Reply:
I think here we would like to limit per-user-per auth method

Reply:
That is a good point! Will enforce per entityID + methodID then.